src/general/Mutations.cs

```diff
@@ -440,6 +440,47 @@ private string MutateWord(string name)
         int letterChanges = 0;
         int changes = 0;
 
+        // Case of 1-letter words, e.g. Primum B
+        if (newName.Length == 1)
+        {
+            var letter = newName.ToString(0, 1);
+            bool isVowel = Vowels.Any(item => item == letter);
```

Comment:
Wouldn't `Contain`? be more natural here than Any with a lambda?

Reply:
I guess it would! However, I reused an instruction from the original part (still there, l.589). So there might have been a precedent reason, or perhaps it just went under the radar? 

I can go along and modify both, as I agree this should be better and not break anything.

Reply:
It's either my fault, or someone elses. When I was porting over the entire codebase to C#, I just tried to do it as fast as a possible without having time (or at the time C# experience) to make everything perfect.

Reply:
Understandable, especially as I can relate for the C# experience haha. I changed this throughout the file for consistency (there might still be occurrences in others though).

I think nonetheless that this could use a proper refactory, as of now I don't even have a fully clear picture of the algorithm, although I think I guess its main lines behaviors. I also think that some parts could be improved as well (early returns when changes limit reached, a `for(int i = 1...) { if (... && i > 1 { ... }  }`... 

I did not make much changes here to not overload the PR, which fixes a game-crashing bug. But I could consider it at some point.

Reply:
I don't also have a clear picture of the algorithm. I just did AngelScript to C# conversion in a really mechanical way.